terraform/modules/ecr/events.tf

```diff
@@ -0,0 +1,38 @@
+resource "aws_cloudwatch_event_rule" "ecr_image_scan_results" {
+  name        = "${var.prefix}-ecr-image-scan-results"

```

Comment:
This might be better referencing the name of the repository rather than a chosen env prefix.